### DISCUSSION TITLE
Will it Support Modrinth?

### DISCUSSION THREAD

**Discussion by mlengka93** [calloatti/modrobot#25] [General]
As the title says. I found this through reddit kinda hard tho, but I love it. Maybe you want to release on CF or something..

**Comment by calloatti** [comment]
No, there is no support for Modrinth currently and it will not be added since it would violate the TOS of using the CurseForge API

**Reply by Bluscream** [reply]
What you're saying makes no sense. Using a second service can not break ToS of another service

**Reply by calloatti** [reply]
https://support.curseforge.com/en/support/solutions/articles/9000207405-curse-forge-3rd-party-api-terms-and-conditions

"Developer shall not use the API or SDK, or any materials obtained through the API or SDK, in order build any product or service that competes, directly or indirectly, with CF, CFC or the Platform."

I chose to respect CF and not integrate a competing service while using their API, please respect my decision. You are of course free to create you own mod updater.

**Reply by Bluscream** [reply]
Luckily better updaters exist already. Can be closed and forgotten about

**Reply by calloatti** [reply]
k
